src/include/test/unit_flat_pq_index.cc

```diff
@@ -441,7 +441,7 @@ TEST_CASE(
 
   SECTION("pq_encoding") {
     auto avg_error = pq_idx.verify_pq_encoding(training_set);
-    CHECK(avg_error < 0.08);
+    CHECK(avg_error < 0.081);
```

Comment:
I don't think that the test values should be fine-tuned for the specific seed. I think that we should set generic expectations in the tests.

Reply:
Yep, makes sense. I changed here though because previously we had a fixed seed - i.e. we used `0xdeadbeef`:
```
sub_kmeans_random_init(training_set, centroids_, sub_begin, sub_end, 0xdeadbeef);
```
And now we have a random seed. So it's more like we previously had tuned it perfectly for a single seed, and now it's random again.